### PR TITLE
Add README.md and update CMakeLists.txt for Skylab v2

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

This PR updated the main branch of jedi-bundle (public) for the Skylab v2 release. This PR should be compared to https://github.com/JCSDA-internal/jedi-bundle/pull/18 for the internal main branch.

This PR should be tested ...

### CODE REVIEW

**Comment by shlyaeva** [comment]
I can test tomorrow morning

**Comment by climbfuji** [comment]
> I can test tomorrow morning

Perfect, thanks. I am testing this (at least with ctest) on my mac now.

File: CMakeLists.txt [hunk 1]
```diff
@@ -25,31 +25,31 @@ ecbuild_bundle_initialize()
...
-ecbuild_bundle( PROJECT saber    GIT "https://github.com/jcsda/saber.git"       TAG 1.3.0 )
+ecbuild_bundle( PROJECT oops     GIT "https://github.com/jcsda/oops.git"        TAG 1.4.0 )
+ecbuild_bundle( PROJECT vader    GIT "https://github.com/jcsda/vader.git"       TAG 1.1.0 )
+ecbuild_bundle( PROJECT saber    GIT "https://github.com/jcsda/saber.git"       TAG 1.4.0 )
 
-ecbuild_bundle( PROJECT crtm     GIT "https://github.com/jcsda/crtm.git"        TAG v2.3-jedi.1 )
-ecbuild_bundle( PROJECT gsw      GIT "https://github.com/jcsda/GSW-Fortran.git" TAG v3.06 )
-ecbuild_bundle( PROJECT ioda     GIT "https://github.com/jcsda/ioda.git"        TAG 2.2.0 )
-ecbuild_bundle( PROJECT ufo      GIT "https://github.com/jcsda/ufo.git"         TAG 1.3.0 )
+ecbuild_bundle( PROJECT crtm     GIT "https://github.com/jcsda/crtm.git"        TAG v2.4-jedi )
```
Comment (srherbener) [inline CMakeLists.txt]: In release/skylab-v2 this line (CRTM) is commented out. Is that supposed to be the same here?

Comment (climbfuji) [reply CMakeLists.txt]: Yes! Good catch, thanks

Comment (climbfuji) [reply CMakeLists.txt]: Fixed in https://github.com/JCSDA/jedi-bundle/pull/1/commits/f365b6be4d8cb840aace43cc2d9b10bcb46950f7

**Review by shlyaeva — APPROVED** [review]
Tested on orion with Intel spack v2 (build + ctest), works as expected except for a few failing tests:

Unexpected: saber spectralb tests fail with:
> 473: [Orion-login-3:79177:0:79177] Caught signal 11 (Segmentation fault: address not mapped to object at address (nil))
473: ==== backtrace (tid:  79177) ====
473:  0 0x000000000005fd5e __ltinv_ctlad_mod_MOD_ltinv_ctlad._omp_fn.0()  /work/noaa/da/role-da/spack-stack/spack-stack-v1/cache/build_stage/spack-stage-ectrans-1.0.0-l2ij5ls75omb5xaksmxasmmhdxwxhie7/spack-src/src/trans/internal/ltinv_ctlad_mod.F90:107
473:  1 0x000000000000fe22 GOMP_parallel()  /home/builder/ktietz/cos6/ci_cos6/ctng-compilers_1622658800915/work/.build/x86_64-conda-linux-gnu/src/gcc/libgomp/parallel.c:171

**Comment by climbfuji** [comment]
> Tested on orion with Intel spack v2 (build + ctest), works as expected except for a few failing tests:
> 
> Unexpected: saber spectralb tests fail with:
> 
> > 473: [Orion-login-3:79177:0:79177] Caught signal 11 (Segmentation fault: address not mapped to object at address (nil))
> > 473: ==== backtrace (tid:  79177) ====
> > 473:  0 0x000000000005fd5e __ltinv_ctlad_mod_MOD_ltinv_ctlad._omp_fn.0()  /work/noaa/da/role-da/spack-stack/spack-stack-v1/cache/build_stage/spack-stage-ectrans-1.0.0-l2ij5ls75omb5xaksmxasmmhdxwxhie7/spack-src/src/trans/internal/ltinv_ctlad_mod.F90:107
> > 473:  1 0x000000000000fe22 GOMP_parallel()  /home/builder/ktietz/cos6/ci_cos6/ctng-compilers_1622658800915/work/.build/x86_64-conda-linux-gnu/src/gcc/libgomp/parallel.c:171

Thanks for testing! This error is obviously unrelated to this PR, looks like a problem with the trans-enabled atlas in saber when OpenMP is used.

**Review by srherbener — APPROVED** [review]

**Pull request merged**